src/dbus_mpris.rs

```diff
@@ -153,7 +159,16 @@ async fn create_dbus_server(
         panic!("Lost connection to D-Bus: {}", err);
     });
 
-    conn.request_name("org.mpris.MediaPlayer2.spotifyd", false, true, true)
+    let path:String;
+    
+    if unique_mpris {
+        path = format!("{a}_{b}", a=DBUS_PATH, b=&device_name);
+    }
+    else{
+        path = DBUS_PATH.to_string();
+    }
```

Comment:
We can avoid allocations here using [`Cow`s](https://doc.rust-lang.org/std/borrow/enum.Cow.html). (Would of course be better to import `std::borrow::Cow` below.)

```suggestion
    let path: std::borrow::Cow<str> = if unique_mpris {
        format!("{DBUS_PATH}_{device_name}").into()
    } else {
        DBUS_PATH.into()
    };
```